content/blog/highlights-of-2023/index.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog-post
+title: Highlights of 2023 
+description: "A Year of Growth: Highlights from 2023 for Haystack and our Community"
+featured_image: thumbnail.png
+images: ["blog/highlights-of-2023/thumbnail.png"]
+featured_image_caption: Highlights of 2023
+alt_image: Highlights of 2023
+toc: True
+date: 2024-01-05
+last_updated:  2024-01-05
+authors:
+  - Bilge Yucel
+tags: ["Open Source", "Haystack 2.0", "Community"]
+---	
+
+> *This post was originally shared through the Haystack newsletter. [Subscribe now](https://landing.deepset.ai/haystack-community-updates) to stay in the loop on all things Haystack, including the latest updates, new features, captivating content, and upcoming exciting events!* 🗞️
+
+What a fantastic year it has been! In 2023, the world of AI saw tremendous progress, making it an exciting time for learning and growth. As we wind down and approach the new year, it's a good moment to take a breather, look back on the past year, and appreciate the highlights before gearing up for what's ahead. We've taken some time to review the noteworthy moments that made 2023 special for Haystack and its community.
+
+A big thank you for being part of this journey! Here's to a wonderful New Year filled with dizzying generative AI and NLP advancements 🥂✨
+
+## 💫 Haystack Highlights
+
+- The biggest highlight of the year was [**Haystack 2.0-Beta**](https://haystack.deepset.ai/blog/introducing-haystack-2-beta-and-advent), which is a more flexible, extendable, customizable version of Haystack v1 💯
+- Haystack surpassed [**12,000 stars**](https://github.com/deepset-ai/haystack) on GitHub 🌟
+- It has been exactly 2 years since the [first stable release](https://github.com/deepset-ai/haystack/releases/tag/v1.0.0) of Haystack 🎉
+- We introduced [Haystack Integrations](https://haystack.deepset.ai/integrations) to highlight community-contributed external packages and incorporate additional technologies like **Amazon**, **VertexAI**, **Elasticsearch**, **OpenSearch**, **Chroma**, **Pinecone**, **Cohere**, **Gradient**, **Jina**, and **Ollama** that seamlessly integrate with Haystack 🧩
+- We hosted the first [**Advent of Haystack**](https://haystack.deepset.ai/advent-of-haystack) 🎄
+- **Over 80 pull requests from the community** were merged 🚀
+- We launched [Haystack Blog](https://haystack.deepset.ai/blog), where we publish articles about Haystack, LLMs, demos, and use cases 📚 **P.S.** Did you know that you can [contribute to Haystack Blog](https://github.com/deepset-ai/haystack-home/pulls)?
+
+![Festive-themed visual with snowman, christmas tree and text displaying 'Haystack 2.0-Beta' and 'Advent of Haystack.](haystack-highlight.png#small "_We announced [Haystack 2.0-Beta](https://haystack.deepset.ai/blog/introducing-haystack-2-beta-and-advent) and hosted the first [Advent of Haystack](https://haystack.deepset.ai/advent-of-haystack)_")
```

Comment:
this image isn't displaying in the preview deploy version of the page

Reply:
Apparently, there's a bug that prevents adding links to image captions. It works locally but not in the deployed version